package.json

```diff
@@ -27,7 +27,9 @@
     "husky": "^7.0.1",
     "lint-staged": "^11.1.2",
     "prettier": "^2.5.1",
-    "prettier-plugin-tailwindcss": "^0.1.6",
+    "prettier-plugin-tailwindcss": "^0.1.6"
+  },
+  "dependencies": {
```

Comment:
It was required for my deployment to work on Heroku.
Heroku was throwing error that turbo is missing during start command